.gitignore

```diff
@@ -1,57 +1,61 @@
+**/*.backup
```

Comment:
Alphabetizing at the same time of adding things was a bad idea here.

for tfstate files, we need to keep them somewhere, but not in this repo, so really I should just drop backup files here. and `.terraform` directories.

Reply:
Yeah, I would avoid checking these in. How to track the state will be a bigger discussion.. probably putting into a project-specific GCS bucket is easiest, but we may want to sync with sysadmin before we go further down this path.

Reply:
Do you think using the cloud version would be a non-starter?

I could see having versioned files in a bucket. That solves two things.